src/researchtikpy/get_videos_hashtag.py

```diff
@@ -53,26 +55,16 @@ def _create_query(hashtags: list[str], region_code: str, music_id: str, effect_i
     return query
 
 
-def get_videos_query(query: dict, access_token: str, start_date: str, end_date: str, total_max_count: int, max_count=100, rate_limit_pause=60) -> pd.DataFrame:
+def get_videos_query(query: dict, access_token: str, start_date: str, end_date: str, total_max_count: int, max_count=100) -> pd.DataFrame:
     """
     Like get_videos_hashtag(), but you can pass a custom `query` object
     For the `query` parameter, see the TikTok API documentation: https://developers.tiktok.com/doc/research-api-specs-query-videos/
     For the rest of parameters, see get_videos_hashtag()
     """
 
-    fields = "id,video_description,create_time,region_code,share_count,view_count,like_count,comment_count,music_id,hashtag_names,username,effect_ids,playlist_id,voice_to_text"
-
-    endpoint = "https://open.tiktokapis.com/v2/research/video/query/"
-    url_with_fields = f"{endpoint}?fields={fields}"
-
-    assert isinstance(access_token, str), "access_token must be a string!"
-    headers = {
-        "Content-Type": "application/json",
-        "Authorization": f"Bearer {access_token}"
-    }
-
     start_date_dt = datetime.strptime(start_date, "%Y%m%d")
     end_date_dt = datetime.strptime(end_date, "%Y%m%d")
+    assert start_date_dt <= end_date_dt, "start_date must be before or equal end_date!"
```

Comment:
Should be an `if` statement (see [assert](https://docs.astral.sh/ruff/rules/assert/))

Reply:
I'll change it to an exception